packages/request-client.js/package.json

```diff
@@ -50,20 +55,25 @@
     "@requestnetwork/epk-signature": "0.5.34",
     "@requestnetwork/multi-format": "0.15.9",
     "@requestnetwork/payment-detection": "0.35.0",
+    "@requestnetwork/payment-processor": "0.37.0",
     "@requestnetwork/request-logic": "0.26.9",
     "@requestnetwork/smart-contracts": "0.28.0",
     "@requestnetwork/transaction-manager": "0.26.9",
     "@requestnetwork/types": "0.35.0",
     "@requestnetwork/utils": "0.35.0",
     "axios": "0.27.2",
+    "cors": "2.8.5",
     "ethers": "5.5.1",
-    "tslib": "2.3.1"
+    "tslib": "2.3.1",
+    "express": "4.18.2",
+    "dotenv": "16.0.3"
```

Comment:
Please remove the cors, tslib, express, and dotenv dependencies (and devDependencies) if they are not needed